src/olympia/devhub/templates/devhub/verify_email.html

```diff
@@ -0,0 +1,52 @@
+{% extends "devhub/base.html" %}
+
+{% set title = _('Email Verification') %}
```

Comment:
I'll leave this comment here, but it applies to a few places below, and in the templates - is it "Email Verification" or "Email Address Verification"?

Reply:
some places say email, others email address.

Reply:
I think this is the only place that has the short version, and that is mostly so it doesn't get clipped in the tab title.

I did look up the [spec](https://www.w3.org/Provider/Style/TITLE.html) and it says up to 64 is fine, so I will make it the long version so it's consistent.

Reply:
some other places say "Could not verify email", etc (rather than "Could not verify email address"

Reply:
Got it. Updated that one too.